src/components/request/Request.tsx

```diff
@@ -48,33 +51,31 @@ const Request = () => {
     requestTxHash: requestTx,
   });
 
-  const isByTransactionParams = useIsByTransactionParams();
-  const { request, error } = useRequestParams(isByTransactionParams);
-
   useEffect(() => {
-    // TODO: would be nice to do something with the error here, like redirect to the homepage
-    if (!error && request) {
-      if (isByTransactionRequest(request)) {
-        setRequestId(
-          client.setActiveRequestByTransaction({
-            chainId: request.chainId,
-            transactionHash: request.transactionHash,
-            eventIndex: request.eventIndex ?? 0,
-          })
-        );
-      } else {
-        setRequestId(
-          client.setActiveRequest({
-            requester: request.requester.trim(),
-            identifier: request.identifier,
-            timestamp: request.timestamp,
-            ancillaryData: request.ancillaryData,
-            chainId: request.chainId,
-          })
-        );
-      }
+    const request: unknown = Object.fromEntries([...searchParams]);
+    let requestByTx = getRequestInputByTransaction(request);
```

Comment:
this will return a valid type if acceptable, or undefined, same with `getRequestInput`